src/ralph/assets/publishers.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import pyhermes
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from ralph.data_center.models.physical import DataCenterAsset
+from ralph.data_center.models.virtual import Cluster
+from ralph.virtual.models import VirtualServer
+
+
+#TODO:: change topic name, shouldn't this be full path or from settings?
+@pyhermes.publisher(topic='auto_txt_record', auto_publish_result=True)
+def publish_data_to_dnsaaas(obj):
+    return obj.publish_data
```

Comment:
https://github.com/allegro/ralph/blob/ng/src/ralph/settings/base.py#L349
is this enough? or you meant only the publisher registration?